squad/frontend/templates/squad/_subscribe.jinja2

```diff
@@ -9,9 +9,9 @@
     <input type="hidden" name="email" value="{{ user.email }}"/>
     <input type="hidden" name="subscription" value="{{ project.id }}"/>
     {% if project.user_subscriptions %}
-        <button class="btn btn-default" type="submit"><i class='fa fa-bell-slash'>&nbsp;</i></button>
+        <button class="btn btn-default" type="submit" data-toggle="tooltip" title="un-subscribe from project"><i class='fa fa-bell-slash'>&nbsp;</i></button>
```

Comment:
`Un-subscribe` should be written as `unsubscribe`.

Reply:
@jscook2345 good catch..thank you :)